src/dialect/mssql/mssql-query-compiler.ts

```diff
@@ -73,6 +74,11 @@ export class MssqlQueryCompiler extends DefaultQueryCompiler {
     this.visitNode(node.column)
   }
 
+  protected override visitMergeQuery(node: MergeQueryNode): void {
+    super.visitMergeQuery(node)
+    this.append(';')
```

Comment:
This is weird.. Is it really required by mssql?

Reply:
I'm afraid it is. Getting mssql errors otherwise.